src/pdfkernel.cpp

```diff
@@ -1295,13 +1295,12 @@ wxPdfDocument::PutFonts()
       NewObj();
       font->SetFileIndex(m_n);
 
-      bool compressed = true;
```

Comment:
Sorry, in method `PutFonts` the member variable `m_compress` **can't be used**. I have to admit that using the local variable `compressed` (that is always set to `true`) is a bit confusing. Actually, it is a leftover from early days of _wxPdfDocument_ development, when I needed it for debugging purposes. Method `WriteUnicodeMap` which writes data to the memory stream `mos` currently **always compresses** the data. Therefore, for cleanup of the code the statements `Out("/Filter /LateDecode");` should be excuted unconditionally.